startup/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
```

Comment:
* I have exactly the same code but in `routes.js` why you changed the file name ?

Reply:
using index.js inside a directory means that you only need to require the directory name when importing the module.

eg: const app = require('../startup')

instead of 

const app = require("../startup/routes")

Reply:
I figured this out and found it quite useful thank you